### ISSUE TITLE
Publish press release 

### ISSUE THREAD

**Issue by lauradereynal** [MozillaFoundation/digital-skills-observatory#27]
Goal: Publish a press release about the project written in collaboration with Gates and DDD

Owner: @KevZawacki
With @secretrobotron 

Related to #25

**Comment by KevZawacki** [comment]
@LauraReynal @secretrobotron 

First draft ready for review [here](https://docs.google.com/document/d/1wUQ7faI4pqxaWhRy6I_lXzNpP_ZqSnIGbpBN0AvegtY/edit).

**Comment by secretrobotron** [comment]
:+1: :+1: :+1:

Made a bunch of edits for you, @KevZawacki.

**Comment by secretrobotron** [comment]
@KevZawacki update: just waiting on confirmation on that quote, and we're ready to go. cc @LauraReynal

**Comment by KalpanaSimhan** [comment]
@secretrobotron checking to see if the press release is ready to go to Gates for approval

**Comment by secretrobotron** [comment]
I believe so. Text and content looks good, and Mark gave us an approval for his quote. Onward!

**Comment by KalpanaSimhan** [comment]
@secretrobotron @KevZawacki @LauraReynal attached is our press release with minor edits from Gates. Once we incorporate their edits, we have approval to disseminate.

[Grant OPP1135213 DSOreleaseblog_Gates Approved Feb 2.docx](https://github.com/MozillaFoundation/digital-skills-observatory/files/113985/Grant.OPP1135213.DSOreleaseblog_Gates.Approved.Feb.2.docx)

**Comment by chanditome** [comment]
Looks good to me. Good job @KalpanaSimhan and team.

**Comment by secretrobotron** [comment]
@KalpanaSimhan @KevZawacki pretty simple changes. :+1: How do we proceed? [Make :rocket: go now](https://www.youtube.com/watch?v=PfnBfcVfFTA)?

**Comment by KevZawacki** [comment]
@secretrobotron We can publish on blog.webmaker.org, for starters, and also tweet the news. Are there other channels you had in mind?

Unfortunately, with two Advocacy campaigns underway (encryption + Open Web Fellows), we don't have the bandwidth to built a media list and conduct active pitching. 

cc @hpauljohnson

**Comment by secretrobotron** [comment]
Yeah, that's alright. Unless @LauraReynal or someone can think of a reason to do otherwise, I think we just want to have it published — no active pitch necessary.

**Comment by KevZawacki** [comment]
@secretrobotron Cool -- just let me know when I should publish.

**Comment by lauradereynal** [comment]
@KevZawacki :whale: :tada: how are things ? 
I wonder if we can publish on our research blog? We already talked about the project, but I still want to publish it. What do you think ? mzl.la/research

**Comment by KevZawacki** [comment]
@LauraReynal Welcome back!

Publishing on the research blog sounds OK to me. Although, I don't have access to the back-end myself.

**Comment by lauradereynal** [comment]
Thanks @KevZawacki it's good to be back :tulip: 
I can do the publication on our research blog. Has this been posted somewhere else already ?

**Comment by chanditome** [comment]
@LauraReynal a link to the research blog? Or do you mean Webmaker?

**Comment by KevZawacki** [comment]
@LauraReynal As far as I know, it hasn't been published elsewhere.

**Comment by secretrobotron** [comment]
@KevZawacki @LauraReynal if it's ok with you guys, I think it would still be nice to have it published on the webmaker blog. We can publish it on the research one regardless.

**Comment by KevZawacki** [comment]
@secretrobotron OK by me! Let me know if you don't have access -- I can help get it published.

**Comment by secretrobotron** [comment]
@KevZawacki done: https://blog.webmaker.org/mozilla-receives-670k-to-explore-digital-skills-learning-in-kenya

Thanks for the help :).
